tests/tests_combination.yml

```diff
@@ -96,7 +104,10 @@
       failed_when: not __result.stat.exists
 
     - name: Check the filter and action in the files output config
-      command: /bin/grep '\*.info;authpriv.none;auth.none;cron.none;mail.none.*{{ __default_system_log }}' '{{ __test_files_conf }}'
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space